### ISSUE TITLE
Social links on Offices and Services don't have alt text

### ISSUE THREAD

**Issue by poojagunturu96** [middlebury/midd-frontend#724]
Social links on Offices and Services don't have alt text. 

Siteimprove report: https://my2.siteimprove.com/Inspector/7333/A11Y/Page?pageId=72160253158&impmd=D97548790CE293BDC08F6D9B84D3E607&conf=a+aa+aria+si&issue=cantTell+failed&wcag=twopointone#/sia-r11/failed/emgwZ3UwdnI=;OWJmOWFlOWE=

Similar to #602.

**Comment by poojagunturu96** [comment]
Resolved in Drupal.
